locale/exchangecalendar/en-US/oofSettings.dtd

```diff
@@ -1,13 +1,13 @@
-<!ENTITY title.oofsettings "Out of Office settings">
+<!ENTITY title.oofsettings "Out of office settings">
```

Comment:
Did you check how this is written in other applications? It seems to be a fixed term that requires capitalization.